### ISSUE TITLE
YYMMDD vs Unix

### ISSUE THREAD

**Issue by AlexBSiteEye** [time-lapse-api/docs#3]
There seems to be a bit of inconsistency between how dates are represented in parts of the API.

For example the "request_time" on responses is always returned as Unix, but other date types can be specified.

Can I propose that we use Unix timestamps as the only date representation? Or if not, that there is a mechanism so the format can be specified on every request for every date?

What are your thoughts?

**Comment by peteredhead** [comment]
Personally, I would prefer to use the ISO8601 date format as the default. 

I believe the optional parameter to request another format is something that was  implemented as a convenience function for our own front-end (before my time). I don't see any need to include this in a public facing version of the API.

**Comment by AlexBSiteEye** [comment]
The only problem I can see with using ISO8601 is that we'd have to URL encode/decode the date string each time as it could contain characters that aren't URL friendly.

Can I suggest we use a similar format without the special characters, maybe 'YYYYMMDDHHmmSS'?

**Comment by peteredhead** [comment]
My only concern with other formats is how the timezone is specified. Unless we mandate that all times are UTC?

**Comment by AlexBSiteEye** [comment]
I hadn't considered timezone. I think in that case using ISO8601 is probably the best idea with properly encoded URL characters.

**Comment by peteredhead** [comment]
I've created a pull request for a proposed solution to this issue.
- All requests for dates are to be ISO8601 date format (eg YYYY-MM-DD)
- All date responses are in ISO8601 date format
- All datetime responses are in ISO8601 datetime format (eg YYYY-MM-DDThh:mm:ss±hh:mm)

As a result, I've removed the two date and time fields from the image response and replaced this with a single datetime.

The timestamp in the response indicating the request time is still in Unix time. If we agree with the above, I'll open a separate issue for that one.
